### ISSUE TITLE
DCAT: Editorial inconsistencies

### ISSUE THREAD

**Issue by andrea-perego** [w3c/dxwg#890]
Going through the spec, I noticed a number of editorial inconsistencies.

I'm going to create a draft PR to fix this. In case I find any editorial issue requiring a decision from the WG, I will report it here.

**Comment by andrea-perego** [comment]
I completed a first round of revisions via a draft PR (https://github.com/w3c/dxwg/pull/894). The list of revisions are documented in the PR, where there's also a link to the HTML preview and diff.

The revisions are, IMO, touching anything controversial. There's only one point I would like to draw your attention on:

The `dcat:*URL` properties (`dcat:accessURL`, `dcat:downloadURL`, `dcat:endpointURL`) have inconsistent labels. Sometimes they use "* URL", sometimes "* address" (e.g., "access URL" / "access address"). 

I modified them to use always the "* URL" pattern - I think that makes they semantics clearer, and it ensures consistency with the naming of the same properties in DCAT 2014.

Anyway, if we decide otherwise, we can always switch to another naming pattern (the main issue, now fixed, is to ensure labels are consistent).
